src/algebraic_topology/simplicial_object.lean

```diff
@@ -232,6 +232,29 @@ variable {C}
 
 end augmented
 
+open_locale simplicial
+
+/-- Aaugment a simplicial object with an object. -/
+@[simps]
+def augment (X : simplicial_object C) (X₀ : C) (f : X _[0] ⟶ X₀)
+  (w : ∀ (i : simplex_category) (g₁ g₂ : [0] ⟶ i),
+    X.map g₁.op ≫ f = X.map g₂.op ≫ f) : simplicial_object.augmented C :=
+{ left := X,
+  right := X₀,
+  hom :=
+  { app := λ i, X.map (simplex_category.const i.unop 0).op ≫ f,
+    naturality' := begin
+      intros i j g,
+      dsimp,
+      rw (show g = g.unop.op, by simp),
```

Comment:
`unop op = id` is a lemma already I hope?

Reply:
It should be `g.unop_op`. But maybe that didn't work??

Reply:
`g.op_unop` doesn't work because the hom in `quiver.hom.op_unop` is implicit :-(
Anyway, I changed this to a rewrite without a simp, in case that makes you more comfortable.

Reply:
ahh, that's a shame :(

Reply:
Fixed in #8088